codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/CustomizableOperationGenerator.kt

```diff
@@ -34,9 +34,9 @@ class CustomizableOperationGenerator(
                 .resolve("CustomizableOperation"),
             "CustomizableSend" to ClientRustModule.Client.customize.toType()
                 .resolve("internal::CustomizableSend"),
-            "HttpRequest" to RuntimeType.smithyRuntimeApi(runtimeConfig)
+            "HttpRequest" to RuntimeType.smithyRuntimeApiClient(runtimeConfig)
                 .resolve("client::orchestrator::HttpRequest"),
-            "HttpResponse" to RuntimeType.smithyRuntimeApi(runtimeConfig)
+            "HttpResponse" to RuntimeType.smithyRuntimeApiClient(runtimeConfig)
```

Comment:
It appears like the types are also available without the `client` feature flag, under the `http` module. I wonder then why we don't use them directly instead of going through the `client` module?

Reply:
Yeah, this makes sense to me. We could probably even get rid of the HttpRequest/HttpResponse types from th eorchestrator module at this point. I think it's not so high priority though, and we can always deprecate them later.